### PR TITLE
Fix xml namespace in org.godotengine.Godot.xml

### LINKED ISSUE #74778: wrong namespace on document element in '/usr/share/mime/packages/org.godotengine.Godot.xml' (should be http://www.freedesktop.org/standards/shared-mime-info)
### Godot version

v4.0.stable.arch_linux (4.0-2)

### System information

Endeavour OS with KDE wayland

### Issue description

After I update my system I got a message
`Wrong namespace on document element in '/usr/share/mime/packages/org.godotengine.Godot.xml' (should be http://www.freedesktop.org/standards/shared-mime-info)`

Here is the full log of the update:
```
[2023-03-11T01:38:23+0000] [PACMAN] Running 'pacman -S -y -u --config /etc/pacman.conf --'
[2023-03-11T01:38:23+0000] [PACMAN] synchronizing package lists
[2023-03-11T01:38:24+0000] [PACMAN] starting full system upgrade
[2023-03-11T01:38:46+0000] [ALPM] running '60-mkinitcpio-remove.hook'...
[2023-03-11T01:38:46+0000] [ALPM] transaction started
[2023-03-11T01:38:46+0000] [ALPM] upgraded eos-bash-shared (23-5 -> 23-7)
[2023-03-11T01:38:46+0000] [ALPM] upgraded girara (0.3.7-1 -> 0.3.9-1)
[2023-03-11T01:38:47+0000] [ALPM] upgraded go (2:1.20.1-1 -> 2:1.20.2-1)
[2023-03-11T01:38:47+0000] [ALPM] installed libspeechd (0.11.4-1)
[2023-03-11T01:38:47+0000] [ALPM] upgraded libsquish (1.15-3 -> 1.15-7)
[2023-03-11T01:38:47+0000] [ALPM] upgraded godot (3.5.1-4 -> 4.0-2)
[2023-03-11T01:38:47+0000] [ALPM] upgraded libnm (1.42.2-1 -> 1.42.4-1)
[2023-03-11T01:38:48+0000] [ALPM] upgraded linux (6.2.2.arch1-1 -> 6.2.2.arch2-1)
[2023-03-11T01:38:49+0000] [ALPM] upgraded linux-headers (6.2.2.arch1-1 -> 6.2.2.arch2-1)
[2023-03-11T01:38:49+0000] [ALPM] upgraded networkmanager (1.42.2-1 -> 1.42.4-1)
[2023-03-11T01:38:49+0000] [ALPM] upgraded zathura (0.5.2-1 -> 0.5.2-2)
[2023-03-11T01:38:49+0000] [ALPM] transaction completed
[2023-03-11T01:38:50+0000] [ALPM] running '30-systemd-daemon-reload.hook'...
[2023-03-11T01:38:50+0000] [ALPM] running '30-systemd-udev-reload.hook'...
[2023-03-11T01:38:50+0000] [ALPM] running '30-systemd-update.hook'...
[2023-03-11T01:38:50+0000] [ALPM] running '30-update-mime-database.hook'...
[2023-03-11T01:38:50+0000] [ALPM-SCRIPTLET] Wrong namespace on document element in '/usr/share/mime/packages/org.godotengine.Godot.xml' (should be http://www.freedesktop.org/standards/shared-mime-info)
[2023-03-11T01:38:50+0000] [ALPM] running '60-depmod.hook'...
[2023-03-11T01:38:51+0000] [ALPM] running '90-mkinitcpio-install.hook'...
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET] ==> Building image from preset: /etc/mkinitcpio.d/linux.preset: 'default'
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET]   -> -k /boot/vmlinuz-linux -c /etc/mkinitcpio.conf -g /boot/initramfs-linux.img
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET] ==> Starting build: 6.2.2-arch2-1
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET]   -> Running build hook: [base]
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET]   -> Running build hook: [udev]
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET]   -> Running build hook: [autodetect]
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET]   -> Running build hook: [modconf]
[2023-03-11T01:38:51+0000] [ALPM-SCRIPTLET]   -> Running build hook: [block]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: xhci_pci
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [keyboard]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [keymap]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [consolefont]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> WARNING: consolefont: no font found in configuration
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [filesystems]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [fsck]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> Generating module dependencies
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> Creating zstd-compressed initcpio image: /boot/initramfs-linux.img
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> Image generation successful
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> Building image from preset: /etc/mkinitcpio.d/linux.preset: 'fallback'
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> -k /boot/vmlinuz-linux -c /etc/mkinitcpio.conf -g /boot/initramfs-linux-fallback.img -S autodetect
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET] ==> Starting build: 6.2.2-arch2-1
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [base]
[2023-03-11T01:38:52+0000] [ALPM-SCRIPTLET]   -> Running build hook: [udev]
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET]   -> Running build hook: [modconf]
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET]   -> Running build hook: [block]
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: wd719x
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: aic94xx
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: qed
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: bfa
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: qla2xxx
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: qla1280
[2023-03-11T01:38:53+0000] [ALPM-SCRIPTLET] ==> WARNING: Possibly missing firmware for module: xhci_pci
[2023-03-11T01:38:54+0000] [ALPM-SCRIPTLET]   -> Running build hook: [keyboard]
[2023-03-11T01:38:55+0000] [ALPM-SCRIPTLET]   -> Running build hook: [keymap]
[2023-03-11T01:38:55+0000] [ALPM-SCRIPTLET]   -> Running build hook: [consolefont]
[2023-03-11T01:38:55+0000] [ALPM-SCRIPTLET] ==> WARNING: consolefont: no font found in configuration
[2023-03-11T01:38:55+0000] [ALPM-SCRIPTLET]   -> Running build hook: [filesystems]
[2023-03-11T01:38:55+0000] [ALPM-SCRIPTLET]   -> Running build hook: [fsck]
[2023-03-11T01:38:56+0000] [ALPM-SCRIPTLET] ==> Generating module dependencies
[2023-03-11T01:38:56+0000] [ALPM-SCRIPTLET] ==> Creating zstd-compressed initcpio image: /boot/initramfs-linux-fallback.img
[2023-03-11T01:38:57+0000] [ALPM-SCRIPTLET] ==> Image generation successful
[2023-03-11T01:38:57+0000] [ALPM] running 'dbus-reload.hook'...
[2023-03-11T01:38:57+0000] [ALPM] running 'eos-reboot-required.hook'...
[2023-03-11T01:38:57+0000] [ALPM-SCRIPTLET] ==> INFO: Reboot is recommended due to the upgrade of core system package(s).
[2023-03-11T01:38:57+0000] [ALPM] running 'gtk-update-icon-cache.hook'...
[2023-03-11T01:38:57+0000] [ALPM] running 'rebuild-detector.hook'...
[2023-03-11T01:38:57+0000] [ALPM] running 'update-desktop-database.hook'...
```

### Steps to reproduce

I am not sure how this affect the engine, but I got this message only when I update from version 3.5 and 4.0.

### Minimal reproduction project

N/A

### PR DESCRIPTION
**Pull request by Paragoumba**
Fixed xml namespace in org.godotengine.Godot.xml.
This fixes #74778.

### CODE REVIEW

File: misc/dist/linux/org.godotengine.Godot.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<mime-info xmlns="https://specifications.freedesktop.org/shared-mime-info-spec">
+<mime-info xmlns="https://www.freedesktop.org/standards/shared-mime-info">
```
[the remark this answers is not in the document]
Comment (Paragoumba) [reply misc/dist/linux/org.godotengine.Godot.xml]: You are right, I committed your changes

**Review by deralmas — APPROVED** [review]
Great! I haven't tested this as I don't have MIME stuff on my laptop and I don't use my desktop that often, but I'm sure that this works.

I wonder why both the error and the docs require HTTP though, I wonder why they haven't switched over to HTTPS yet.

Anyways, I'm sure that this will work fine, so LGTM!

**Comment by Paragoumba** [comment]
The https version works with a web browser, that's why I had put it at first. I have tested to edit the file with both protocols and it only works with http. Might be worth investigating to see if https might work in the future!

**Comment by YuriSizov** [comment]
Can you please amend your commit message to be more descriptive? Something like the title of this PR would work great, "Fix xml namespace in org.godotengine.Godot.xml"

**Comment by Paragoumba** [comment]
@YuriSizov I amended my commit and rebased on top of master

**Review by akien-mga — APPROVED** [review]

**Comment by YuriSizov** [comment]
Thanks! And congrats on your first merged Godot pull request!

**Comment by YuriSizov** [comment]
Cherry-picked for 4.0.2.

**Pull request merged**